src/WCSimPMTObject.cc

```diff
@@ -220,7 +219,27 @@ G4float PMT20inch::GetmaxQE(){
   return maxQE;
 }
 
+// Should be actual PMT Dark Rate, not effective dark rate in detector including other LE noise
+G4float PMT20inch::GetDarkRate(){
+  /* From e-mail discussion with A.Konaka and S.Nakayama:
+   * SK-I: 4.2 kHz 
+   * SK-IV:5.7 kHz, both before electronics threshold in skdetsim
+   * Measured DN with 0.25 pe threshold:
+   * SK-I: 3.4 kHz  (2003 SK-NIM: 3 kHz. A.Konaka: "2kHz with hot PMTs removed?") 
+   * SK-IV: 4.5 kHz (higher due to FRP)
+   * ToDo: investigate after updating electronics routing, whether to change value to 3.4 kHz
+   */
+
```

Comment:
If SKDETSIM has 4.2kHZ after the 0.25 pe threshold as Okajima-san showed, then this is inconsistent with the e-mail discussion that I summarized in the comments, as it should have been before (3-3.4 kHz in the 2003 SK-NIM). Current behaviour is consistent with SKDETSIM, but I would prefer to leave the comments in as I'm not sure whether this is the correct behaviour for both. Please correct if I didn't understand/summarize the e-mail discussion properly.
Changing this number is for a separate pull request, but this comments makes sure we don't forget about it.